### DISCUSSION TITLE
Feature request: Display user's local time in their profile page (based on their location)

### DISCUSSION THREAD

**Discussion by sequba** [community/community#12595] [Other Feature Feedback, Questions, & Ideas]
I collaborate with people from all over the world on GH. Often I need to check the local time for some users to learn whether they might be asleep and what time can I expect them to answer my messages. Displaying that information next to the user's location in the profile page would be a nice feature.

**Comment by NikhilMath** [comment]
I think that would be a great idea, or we can also use UTC as well, (maybe that would be more coding friendly)
